frontend/www/js/omegaup/components/arena/NewClarificationPopup.vue

```diff
@@ -55,7 +55,11 @@
       </div>
       <div class="form-group row">
         <div class="col-sm-10">
-          <button type="submit" class="btn btn-primary">
+          <button
+            type="submit"
+            class="btn btn-primary"
+            :disbaled="!shouldSubmitClarification"
```

Comment:
```suggestion
            :disabled="!canSubmitClarification"
```

en este caso "should" es "debería", pero lo que queremos ver es si "se puede".

Reply:
Listo